### PR TITLE
[REVIEW] Add MNMG Linear Regression simple fit

### PR DESCRIPTION
**Pull request by Nanthini10**
Adds a simple test for MNMG linear regression model to ensure it doesn't break during release

Closes https://github.com/rapidsai/cuml/issues/2444

### CODE REVIEW

**Comment by mike-wendt** [comment]
@aleksficek can you work with @msadang and add this test to a RAPIDS devel container and run the test like we do for https://gpuci.gpuopenanalytics.com/job/rapidsai/job/nightly-tests/job/docker-test-rapids-integration/ ?

If the new test pass can you add your review here and let me know?

**Comment by aleksficek** [comment]
rerun tests

**Comment by JohnZed** [comment]
It is possible to run this on one GPU or several. Right now, we have the binary-only cumlprims library, which is not exercised at all in integration tests. In the last release, we saw a failure where the final cumlprims build was not built for all GPU architectures, so it completely failed on  pascal, but we had not run it against the full matrix of systems to test. That sort of error will be easy to catch with a quick test like this.

**Comment by dillon-cullinan** [comment]
Thanks for the clarification @JohnZed

File: test/cuml/test_mnmg.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    cluster = LocalCUDACluster(threads_per_worker=1)
+    _ = Client(cluster)
+
+    # Create and populate a GPU DataFrame
+    df_float = cudf.DataFrame()
+    df_float['0'] = [1.0, 2.0, 5.0]
+    df_float['1'] = [4.0, 2.0, 1.0]
+    df_float['2'] = [4., 2, 1]
+
+    ddf_float = dask_cudf.from_cudf(df_float, npartitions=2)
```
[the remark this answers is not in the document]
Comment (mike-wendt) [reply test/cuml/test_mnmg.py:18]: @JohnZed does this need a change on CI side or just in the script?

[the remark this answers is not in the document]
Comment (Nanthini10) [reply test/cuml/test_mnmg.py:18]: @mike-wendt Just in the script. I'm updating this now

**Comment by mike-wendt** [comment]
Waiting for new PR testing to verify matrix and then will merge

**Comment by mike-wendt** [comment]
rerun tests

**Comment by mike-wendt** [comment]
There are failures [here](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/integration/job/prb/job/rapids-integration-tests/7/) that are related to cugraph - that said I don't [see](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/integration/job/prb/job/rapids-integration-tests/7/CUDA_VER=10.2,LINUX_VER=ubuntu16.04,PYTHON_VER=3.7,RAPIDS_VER=0.15,RUNTIME_DOCKER_REPO=rapidsai%2Frapidsai-nightly/testReport/integration.test.cuml/) this new test being run for cuml

**Comment by mike-wendt** [comment]
@Nanthini10 can you review https://github.com/Nanthini10/integration/blob/branch-0.15/ci/test/cuml.sh to see why your new test is not being run?

**Comment by mike-wendt** [comment]
There's an issue where your new test is not being tested or picked up, looking into it before merging/approving

**Comment by mike-wendt** [comment]
I see the [test](https://gpuci.gpuopenanalytics.com/job/rapidsai/job/integration/job/prb/job/rapids-integration-tests/9/CUDA_VER=10.2,LINUX_VER=ubuntu16.04,PYTHON_VER=3.7,RAPIDS_VER=0.15,RUNTIME_DOCKER_REPO=rapidsai%2Frapidsai-nightly/testReport/junit/test.cuml/) now

**Pull request merged**